srcalc/src/test/java/gov/va/med/srcalc/web/controller/admin/NewVarControllerIT.java

```diff
@@ -122,4 +129,53 @@ public final void testNewMultiSelectNoOptions() throws Exception
                 .param("groupId", "1"))
             .andExpect(model().attributeHasErrors(NewVarController.ATTRIBUTE_VARIABLE));
     }
+    
+    @Test
+    public final void testNewDiscreteNumericalValid() throws Exception
+    {
+        final String key = "testNewMsVarValidKey";
```

Comment:
I think you meant to have the key string reference a discrete numerical instead of multi select.